framework/src/samplers/Sampler.C

```diff
@@ -67,9 +67,7 @@ Sampler::Sampler(const InputParameters & parameters)
     _perf_get_global_samples(registerTimedSection("getGlobalSamples", 1)),
     _perf_get_local_samples(registerTimedSection("getLocalSamples", 1)),
     _perf_get_next_local_row(registerTimedSection("getNextLocalRow", 1)),
-    _perf_advance_generator(registerTimedSection("advanceGenerators", 2)),
```

Comment:
Oops - you took this line out, but didn't take out the declaration.

Reply:
No, just the comma